cli/tools/test.rs

```diff
@@ -1600,16 +1600,18 @@ impl TestEventSender {
         | TestEvent::StepResult(_, _, _)
         | TestEvent::UncaughtError(_, _)
     ) {
-      self.flush_stdout_and_stderr();
+      self.flush_stdout_and_stderr()?;
```

Comment:
Do you know why the panic occurred in the first place? I wonder if converting this to an error will hide an underlying problem. I think probably not a big deal and this is ok? I can't remember this code that well though 😅

Reply:
> Do you know why the panic occurred in the first place?

No clue

> I wonder if converting this to an error will hide an underlying problem.

Yea thats one of my concerns as well; but i also think its not a big deal

Reply:
There are several issue reported like this - they are mostly caused by piping output of `deno` commands to another program (like `less` or `grep`). https://github.com/denoland/deno/pull/17152 was an attempt to solve it for all prints in the code base.

Reply:
I think something is supposed to wait for all this code to finish before tearing down to ensure the pipes have something to send to, but maybe waiting is impractical. It's maybe better to have reports of some test output not displaying (which might now be surfaced somewhere as an error) instead of a panic. I'll approve the PR.